Makefile

```diff
@@ -73,7 +74,7 @@ $(BUILD_DIR)/$(PROJECT): $(STATIK_FILES) $(GO_FILES) $(BUILD_DIR)
 .PHONY: install
 install: $(BUILD_DIR)/$(PROJECT)
 	mkdir -p $(GOPATH)/bin
-	cp $(BUILD_DIR)/$(PROJECT) $(GOPATH)/bin/$(PROJECT)
+	cp $(BUILD_DIR)/$(PROJECT) $(GOBIN)/$(PROJECT)
```

Comment:
I'm sneaking this in because not being able to control where `make install` puts the binary was annoying. Changing `GOPATH` isn't really feasible as that means go build will start pulling down dependencies in that location so honoring `GOBIN` seemed like the natural way to go.

Now one can do `GOBIN=mypath make install` similar to how one can do `GOBIN=mypath go install` and the default behavior remains unchanged. Let me know if this is problematic somehow and I'll remove it.

Reply:
@tejal29 @nkubala any insights?

Reply:
@yhrn it would be great if you can create this change in another PR.